lib/matplotlib/offsetbox.py

```diff
@@ -1053,13 +1055,22 @@ def set_bbox_to_anchor(self, bbox, transform=None):
 
     @_compat_get_offset
     def get_offset(self, bbox, renderer):
-        # docstring inherited
```

Comment:
The docstring for this method is still inherited from the parent class, so this comment should not be removed.